php/admin/class-onboarding.php

```diff
@@ -0,0 +1,392 @@
+<?php
+/**
+ * User onboarding.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license   http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Admin;
+
+use Block_Lab\Component_Abstract;
+use Block_Lab\Blocks\Block;
+
+/**
+ * Class Onboarding
+ */
+class Onboarding extends Component_Abstract {
+
+	/**
+	 * Option name.
+	 *
+	 * @var string
+	 */
+	public $option = 'block_lab_example_post_id';
+
+	/**
+	 * Register any hooks that this component needs.
+	 */
+	public function register_hooks() {
+		add_action( 'current_screen', array( $this, 'admin_notices' ) );
+	}
+
+	/**
+	 * Runs during plugin activation.
+	 */
+	public function plugin_activation() {
+		$this->add_dummy_data();
+		$this->prepare_welcome_notice();
+	}
+
+	/**
+	 * Prepare onboarding notices.
+	 */
+	public function admin_notices() {
+		$example_post_id = get_option( $this->option );
+
+		if ( ! $example_post_id ) {
+			return;
+		}
+
+		$screen = get_current_screen();
+		$slug   = $this->plugin->block_post->slug;
+
+		if ( ! is_object( $screen ) ) {
+			return;
+		}
+
+		$post_id = filter_input( INPUT_GET, 'post', FILTER_SANITIZE_NUMBER_INT );
+
+		/**
```

Comment:
Maybe the `/**` could instead be `/*` 

I think the `/**` at the top of a comment is only for DocBlocks.

Reply:
TIL!

Reply:
Fix in e5a6585.